cli/src/commands/contract/commands/create.ts

```diff
@@ -28,6 +28,7 @@ export default (opts: BaseCommandOptions) => {
     '--admission-webhook-secret [string]',
     'The admission webhook secret is used to sign requests to the webhook url.',
   );
+  command.option('--suppress-warnings', 'This flag suppresses the composition warnings.');
```

Comment:
`This flag suppresses any warnings produced by composition.`

Reply:
Done